File: DotNet/CesiumLanguageWriter/Advanced/ICesiumCartesian2ListValuePropertyWriter.cs
using System.Collections.Generic;

namespace CesiumLanguageWriter.Advanced
{
    /// <summary>
    /// A writer that can write a value as a list of two-dimensional Cartesian values.
    /// </summary>
    public interface ICesiumCartesian2ListValuePropertyWriter : ICesiumPropertyWriter
    {
        /// <summary>
        /// Writes the value as a list of two-dimensional Cartesian values.
        /// </summary>
        /// <param name="values">The values.</param>
        void WriteCartesian2(IEnumerable<Rectangular> values);
    }
}

File: Java/CesiumLanguageWriter/translatedSrc/cesiumlanguagewriter/advanced/ICesiumCartesian2ListValuePropertyWriter.java
package cesiumlanguagewriter.advanced;


import agi.foundation.compatibility.*;
import cesiumlanguagewriter.*;

/**
 * A writer that can write a value as a list of two-dimensional Cartesian values.
 */
@SuppressWarnings( {
        "unused",
        "deprecation",
        "serial"
})
public interface ICesiumCartesian2ListValuePropertyWriter extends ICesiumPropertyWriter {
    /**
    * Writes the value as a list of two-dimensional Cartesian values.
    * @param values The values.
    */
    void writeCartesian2(Iterable<Rectangular> values);
}